module/VuFind/tests/unit-tests/src/VuFindTest/ILS/Driver/DAIATest.php

```diff
@@ -49,93 +49,97 @@ class DAIATest extends \VuFindTest\Unit\ILSDriverTestCase
-                'storagehref' => '',
-                'item_notes' => [],
-                'services' => [],
-                'is_holdable' => false,
-                'addLink' => false,
-                'holdtype' => null,
-                'addStorageRetrievalRequestLink' => false,
-                'customData' => [],
-                'limitation_types' => [],
-                'doc_id' => 'http://uri.gbv.de/document/opac-de-000:ppn:027586081'
-            ],
+        'holdings' => [
+            0 =>
+                [
+                    'status' =>    '',
+                    'availability' => true,
+                    'duedate' => null,
+                    'requests_placed' => '',
+                    'id' => "027586081",
+                    'item_id' => "http://uri.gbv.de/document/opac-de-000:epn:711134758",
+                    'ilslink' => "http://opac.example-library.edu/loan/REQ?EPN=711134758",
+                    'number' => 1,
+                    'barcode' => "1",
+                    'reserve' => "N",
+                    'callnumber' => "ABC 12",
+                    'location' => 'Example Library for DAIA Tests',
+                    'locationid' => 'http://uri.gbv.de/organization/isil/DE-000',
+                    'locationhref' => 'http://www.example-library.edu',
+                    'storage' => 'Abteilung III',
+                    'storageid' => '',
+                    'storagehref' => '',
+                    'item_notes' => [],
+                    'services' => ['presentation', 'loan'],
+                    'is_holdable' => false,
+                    'addLink' => false,
+                    'holdtype' => null,
+                    'addStorageRetrievalRequestLink' => true,
+                    'customData' => [],
+                    'limitation_types' => [],
+                    'doc_id' => 'http://uri.gbv.de/document/opac-de-000:ppn:027586081'
+                ],
+            1 =>
+                [
+                    'status' => '',
+                    'availability' => true,
+                    'duedate' => null,
+                    'requests_placed' => '',
+                    'id' => "027586081",
+                    'item_id' => "http://uri.gbv.de/document/opac-de-000:epn:711134766",
+                    'ilslink' => "http://opac.example-library.edu/DB=1/PPNSET?PPN=027586081",
+                    'number' => 2,
+                    'barcode' => "1",
+                    'reserve' => "N",
+                    'callnumber' => "DEF 34",
+                    'location' => 'Example Library for DAIA Tests',
+                    'locationid' => 'http://uri.gbv.de/organization/isil/DE-000',
+                    'locationhref' => 'http://www.example-library.edu',
+                    'storage' => 'Abteilung III',
+                    'storageid' => '',
+                    'storagehref' => '',
+                    'item_notes' => ['mit Zustimmung'],
+                    'services' => ['presentation', 'loan'],
```

Comment:
It's not clear, to which service the item_note is related. Maybe we need generic item_notes and in contrast to those new service-related item_notes?

Reply:
Could this also be handled by appending/prepending service-specific text to the notes in the general field? (I don't mind adding something new, but if we can keep the ILS driver spec simpler and less specific without losing functionality, that's probably preferable).

Reply:
Yes, I thought about that as well and it may be a workaround if we do not want to introduce a new field .

Reply:
I'll be interested to hear what @hajoseng has to say! The other consideration with either solution is that, since translation for 8.0 is already underway, we don't have the ability to add new translation strings right now... (Though in this case, if it's absolutely necessary, I could live with adding new strings in just English and German for now, since I don't think DAIA is widely used in other language communities yet... but correct me if I'm wrong about that!)